app/templates/components/nypr-brick-item.hbs

```diff
@@ -28,4 +28,5 @@
         itemTitle=item.attributes.title}}
     {{/if}}
   </div>
-</a>
+  <a class="item__link" href="{{item.attributes.url}}"></a>
```

Comment:
this isn't great. `<a>` tags should have nested content AFAIK. but I see what you're trying to do. Try thinking about this component differently. Instead of stretching out a "click zone", split this markup into two sections, something like this...

```handlebars
<div class="brick-item">
  <a class="brick-item__backdrop" style="background-image: url({{item.attributes.imageMain.url}});">
    <div class="item__icons">
      ...
   </div>
  </a>

  <div class="brick-item__head">
    <a class="brick-item__brand" href="{{item.attributes.headers.brand.url}}"> ... </a>
    <h2 class="brick-item__headline><a href="{{item.attributes.url}}"> .. </a></h2>
    {{#if item.audio}} ... {{/if}}
  </div>
</div>
```

So `.brick-item__backdrop` would get the sizing rules currently applied to `.brick_row__item` and `.brick-item__head` would get absolutely positioned at the bottom of the container. This way the backdrop element serves as a link zone "beneath" the title area content.

Also, what's up with `.item__img`?